helm_deploy/values-cp-dev.yaml

```diff
@@ -10,6 +10,28 @@ generic-service:
     enabled: true
     host: prison-api-dev.prison.service.justice.gov.uk
     tlsSecretName: prison-api-cert
+    modsecurity_enabled: true
+    # Missing User Agent Header - 920320
+    # Request Missing an Accept Header - 920300
+    # Restrict file extension - 920440
+    # SQL Injection Attack: Common Injection Testing Detected - 942110
+    # Found User-Agent associated with scripting/generic HTTP client - 913101
+    modsecurity_snippet:
+      SecRuleEngine On
+      SecDefaultAction "phase:2,pass,log,tag:github_team=dps-tech"
+      SecRuleRemoveById 920320
+      SecRuleRemoveById 920300
+      SecRuleRemoveById 920440
+      SecRuleRemoveById 942110
+      SecRuleRemoveById 913101
+      SecRuleUpdateTargetById 932150 "!ARGS:/keywords/"
+      SecRuleUpdateTargetById 942330 "!ARGS:/keywords/"
+      SecRuleUpdateTargetById 942410 "!ARGS:/outcomeComment/"
+      SecRuleUpdateTargetById 942150 "!ARGS:/outcomeComment/"
+      SecRuleUpdateTargetById 942130 "!ARGS:/text/"
+      SecRuleUpdateTargetById 942130 "!ARGS:/.*comment/"
```

Comment:
according to mod security docs this is a regular expression so can exclude arguments that end with `comment` in this way (which hopefully matches the `EndsWith` azure WAF syntax)